### ISSUE TITLE
Documentation of line chart lacks info on registering AnnotationPlugin

### ISSUE THREAD

**Issue by aljoshakoecher** [valor-software/ng2-charts#1406]
I tried to setup a line chart as described in https://valor-software.com/ng2-charts/#LineChart.
Unfortunately, I couldn't get the chart to render as there was a problem with the annotation plugin. The chart was only displayed once I removed the annotation.
I then discovered that the AnnotationPlugin needs to be registered as described here https://www.chartjs.org/chartjs-plugin-annotation/latest/guide/integration.html. The line `Chart.register(AnnotationPlugin)` did it for me. 

This should be added to the example in the docs, or what do you think?

**Comment by asad-c** [comment]
@aljoshakoecher Definitely worth adding, I've just spent hours trying to get it to work until I saw your post.

**Comment by AndeYashwanth** [comment]
Yeah I faced the same issue and spent hours figuring out untill I saw the documentation page of that plugin.
